donut/modules/directory_search/helpers.py

```diff
@@ -12,7 +12,7 @@ def get_hidden_fields(viewer_name, viewee_id):
     """
     if viewer_name is not None:
         is_me = get_user_id(viewer_name) == viewee_id
-        if is_me or Permissions.ADMIN in get_permissions(viewer_name):
+        if is_me or 3 in get_permissions(viewer_name):
```

Comment:
Why use a magic number here?

Reply:
I don't really have a great solution for that. I don't think we should store a massive enum of all permissions, because then we are just replicating the database with python. Does it make sense for each module that uses permissions to maintain its own enum? Either way I was just stuffing that in there as a placeholder to make sure the directory search tests still pass

Reply:
Do we even need a permissions table? It seems like we could just store all the integer permissions values in a `IntEnum` and insert those values directly into `permission_id` columns.

Reply:
I suppose that's possible too. Let's discuss with the whole team on Friday.

Reply:
We decided at the meeting to have a global enum for default (i.e. non module specific) permissions and having modules define permissions enums as needed.

Reply:
Hmm, so do we still need the permissions tables?